app/assets/src/components/views/report/utils/download.js

```diff
@@ -65,7 +65,8 @@ const getDownloadLinkInfoMap = (sampleId, pipelineRun) => ({
     newPage: true,
   },
   [PIPELINE_VIZ_LABEL]: {
```

Comment:
Shouldn't we remove the entry if there is nothing to download?
Link won't be disabled right?